main.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def get_team_id_season_id(url: str) -> list:
+    try:
+        response = requests.get(url)
+    except requests.exceptions.Timeout:
+        print("Request Timed Out.. Please Check Internet Connection Or Use A Valid Proxy..")
+        return ["-1", "-1"]
```

Comment:
Rather than returning -1 -1, you could return None, and check in the caller function

```
if returned_value is None:
    do something special because you know it errored
```

Reply:
if you do that then you can make the type 

Optional[list]